scheds/rust/scx_tickless/src/main.rs

```diff
@@ -135,24 +131,46 @@ impl<'a> Scheduler<'a> {
             warn!("nohz_full is not enabled in the kernel");
         }
 
+        // Generate the list of available CPUs sorted by capacity in descendind order.
+        let mut cpus: Vec<_> = topo.all_cpus.values().collect();
```

Comment:
On AMD CPUs with X3D cores are they clocked lower than regular cores (I don't have any X3D CPUs)? Otherwise, this seems correct.

Reply:
Yeah I also don't have any AMD with X3D vcache. For what I understand the only difference is that the CPUs capacity can change depending if the user selects cache mode vs frequency mode. When it happens we need to restart the scheduler to re-generate the sorted cpus array. It'd nice to have a mechanism that does that automatically, I'll take a look at this next.

Reply:
Zen4 X3D CCD0 has a lower frequency then CCD1

Reply:
> Zen4 X3D CCD0 has a lower frequency then CCD1

Just to make sure I understand, that means the CPUs on CCD0 have lower frequency then the CPUs on CCD1?

In that case everything should work because the frequency determines the CPU capacity and CPUs are sorted by capacity in descending order, so the scheduler will prioritize those in CCD1.

Reply:
> It'd nice to have a mechanism that does that automatically, I'll take a look at this next.

@arighi I am working on something that should provide RCU-style access to pref-core data as it is updated by listening to netlink events in this PR:

- https://github.com/sched-ext/scx/pull/2400